From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

These tests are buggy, relying on missing imports and failing due to
"golden" tests with out-of-order elements.

diff -urN dhall-docs-1.0.7/dhall-docs.cabal dhall-docs-1.0.7-r1/dhall-docs.cabal
--- dhall-docs-1.0.7/dhall-docs.cabal	2021-11-06 07:44:07.490881172 -0600
+++ dhall-docs-1.0.7-r1/dhall-docs.cabal	2021-11-06 08:01:52.500881689 -0600
@@ -127,6 +127,7 @@
     Default-Language: Haskell2010
 
 Test-Suite tasty
+    Buildable: False
     Type: exitcode-stdio-1.0
     Hs-Source-Dirs: tasty
     Main-Is: Main.hs
